### PR TITLE
Update Stratos section

### LINKED ISSUE #282: Deploy Stratos on EKS
There's currently no information on how this is done.

During testing by Prabal he is hitting https://bugzilla.suse.com/show_bug.cgi?id=1117173&GoAheadAndLogIn=1

### LINKED ISSUE #351: Document that Stratos to not use port 80 by default when using load balancers.
Reported internally some time ago during a CaaSP upgrade, afterwards CAP stopped responding.

Since there's a bit of a race condition at play, it looks like Stratos came up first and grabbed port 80. The operator changed the NodePort of susecf-console-ui-ext to be on port 8080 instead of port 80 and everything worked again. 

Suggestion: have the Stratos installation documentation updated to prevent this by setting the http traffic to a port different than port 80, as it already sets the https traffic to port 8443.

### PR DESCRIPTION
**Pull request by btat**
- Deployment on EKS
- Deployment on AKS
- Specify different port (not 80) when using LB

Resolves #282 
Resolves #351

### CODE REVIEW

File: xml/cap_depl_stratos.xml [hunk 1]
```diff
@@ -134,6 +134,99 @@ console-0                         3/3    Running  0         1h
...
+    --set services.loadbalanced=true \
+    --set console.service.http.servicePort=8080
+</screen>
+  <para>
+    Monitor progress:
+  </para>
+<screen>$ watch -c 'kubectl get pods --namespace stratos'</screen>
+
+  <para>
+   Obtain the IP of the service exposed through the public load balancer:
```
Comment (cjschroder) [inline xml/cap_depl_stratos.xml]: IP address, plz :)

File: xml/cap_depl_stratos.xml [hunk 2]
```diff
@@ -134,6 +134,99 @@ console-0                         3/3    Running  0         1h
...
+  </para>
+<screen>&prompt.user;kubectl get service susecf-console-ui-ext --namespace stratos</screen>
+  <para>
+   Use this IP to create an A record.
```
Comment (cjschroder) [inline xml/cap_depl_stratos.xml]: and again!

File: xml/cap_depl_stratos.xml (hunk 1 above)
Comment (btat) [reply xml/cap_depl_stratos.xml]: Fixed in https://github.com/SUSE/doc-cap/pull/376/commits/a188987c5b40161242121bd909b59ab422a7e9aa

File: xml/cap_depl_stratos.xml (hunk 2 above)
Comment (btat) [reply xml/cap_depl_stratos.xml]: Fixed in https://github.com/SUSE/doc-cap/pull/376/commits/a188987c5b40161242121bd909b59ab422a7e9aa

**Comment by cjschroder** [comment]
:+1: +1

**Pull request merged**